app/controllers/dor_controller.rb

```diff
@@ -27,7 +34,7 @@ def delete_from_index
 
   private
 
-  attr_reader :indexer
+  attr_reader :cocina_with_metadata
```

Comment:
How about making `reindex_pid` a function that takes `cocina_with_metadata` as a parameter rather than having an ivar.  This removes any state related bugs.

Reply:
sure, i was on the fence about that, will change